deeplake/core/vector_index/factory.py

```diff
@@ -0,0 +1,37 @@
+from .indexer import Indexer
+from .mutable_indexer import MutableIndexer
+
+from typing import Callable, Dict, Tuple, Any, Optional, Iterable
+
+
+class IndexFactory:
```

Comment:
I really like the way this class is implemented, but I don't get why is it here? The index is created and maintained in indra, so this class should in python wrapper of indra isn't it?